xwiki-platform-distribution/xwiki-platform-distribution-war-dependencies/pom.xml

```diff
@@ -300,7 +300,7 @@
     </dependency>
     <dependency>
       <groupId>org.xwiki.rendering</groupId>
```

Comment:
Wrong groupid.

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/commit/ed0cd6cbbef0be26342da4722244e60a9544f422 Thanks :+1: